MyHeartCounts/Onboarding/AppOnboardingFlow.swift

```diff
@@ -0,0 +1,96 @@
+//
+// This source file is part of the My Heart Counts iOS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+@_spi(TestingSupport)
+import SpeziAccount
+import SpeziFirebaseAccount
+import SpeziHealthKit
+import SpeziNotifications
+import SpeziOnboarding
+import SpeziStudy
+import SwiftUI
+
+// TODO(@lukas) it's a btit absurd that, if the user was logged in before deleting and reinstallng the app, we have an onboarding
+// step showing the account (incl the user's first&last name) and offering a big beautify stay logged in button, only to then in the next
+// step (the consent) ask the user to please manually enter their first&last name...
```

Comment:
We might want to add some nice functionality for this in the future were we can pre-pass context to the consent document rendering mechanism?